Bio/Align/stockholm.py

```diff
@@ -236,10 +236,6 @@ def _store_per_file_annotations(alignment, gf, rows):
                 value = value.pop()
             if key in AlignmentIterator.gf_mapping:
                 alignment.annotations[AlignmentIterator.gf_mapping[key]] = value
```

Comment:
Should we raise a ValueError here if key is not found in gf_mapping?

If not, you can use
```python
try:
    alignment.annotations[AlignmentIterator.gf_mapping[key]] = value
except KeyError:
    pass
```
to avoid having to look up the key twice in gf_mapping.